serverless/knative_eventing/serverless-sinkbinding.adoc

```diff
@@ -0,0 +1,12 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-sinkbinding"]
+= Using SinkBinding
+include::modules/common-attributes.adoc[]
+:context: serverless-sinkbinding
+
+toc::[]
+
+SinkBinding is used to connect event producers, or event sources, to an event consumer, or event sink, for example, a Knative service or application.
```

Comment:
Should event sources and event sink by marked-up, as introducing new terms? 
connect event producers, or _event sources_, to an event consumer, or _event sink_,